src/components/TimePicker24Hour/TimePicker24Hour.jsx

```diff
@@ -39,10 +51,18 @@ const TimePicker24Hour = (props) => {
     const displayTime = e.target.value;
     const newTime = moment(displayTime, "HH:mm").format("HH:mm");
     if (isValidTime(newTime)) {
-      setWarning(false);
+      if (customValidation) {
+        customValidation(newTime);
+      } else {
+        setWarning(false);
```

Comment:
I guess we need to invoke actionForInvalidTime(false) after line 57 to indicate parent that the time is valid.